index.html

```diff
@@ -0,0 +1,243 @@
+                                <li><a class="nav_link black" href="#"><i class="fa fa-tag"></i> Biotechnologie</a></li>
+                                <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i> VisVires New Protein</a></li>
+                            </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>BREAKING: Food testing startup SwissDeCode nets $4.2m from VisVires New Protein, EIT Food on AgFunderNews</p>
+                        </div>
+                    </div>
+                    <div class="cflex fund sfund">
+                        <img class="mb2" alt="ff" src="img/third.png">
+                        <div class="mb2">
+                         <ul class="none">
+                             <li><a class="nav_link black" href="#"><i class="fa  fa-tag"></i>Internet</a></li>
+                            <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i>Buyo Capital</a></li>
+                          </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>SoftBank-backed travel platform Klook raises $200M amid COVID-19 on TechCrunch</p>
+                        </div>
+                    </div>
+                    <div class="cflex fund sfund">
+                        <img class="mb2" alt="ff" src="img/fourth.png">
+                        <div class="mb2">
+                            <ul class="none">
+                                <li><a class="nav_link black" href="#"><i class="fa fa-tag"></i>Mobile</a></li>
+                                <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i>DST Global</a></li>
+                             </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>Food delivery startup Wolt raises $530M after tripling revenues on SiliconANGLE</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+        </main>
+        <!--Footer-->
+        <footer class="cflex">
+            <div class="upperf cflex acenter jcenter">
+                <div class="socials flex sjaround">
+                    <button type="submit" class="bicon"><i class="fa fa-facebook-square"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-youtube"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-twitter"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-instagram"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-pinterest"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-facebook"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-envelope"></i></button>
+                </div>
```

Comment:
## Improve the footer styles:- the social media icons are still not styled properly.

- [ ] Give the appropriate width and height for the social media links so the footer links match the (original website)[https://perma.cc/M5ZV-Q2D6]. 
- [ ] match the appropriate background color of the website for these links.
- [ ] Links should give an impression that they can be clicked. As you can see in the (Source Page)[https://perma.cc/M5ZV-Q2D6] the links change the cursor type when a user hovers over them.